### PR TITLE
Custom Node URLs

### PR DESCRIPTION
**Pull request by staylorwr**
https://www.loom.com/share/43fe5c72b8ec470ab00da989229792ca

### CODE REVIEW

**Review by caleboller — CHANGES_REQUESTED** [review]
Hey @staylorwr - I pulled this down and attempted to test it out, but it's not working as smoothly as your video. New nodes are not being persisted as best I can tell.

https://www.loom.com/share/cad0b8bf9e824f99bf55b53434e17824

File: src/views/Settings/SettingsSelectNode.vue [hunk 1]
```diff
@@ -17,15 +17,22 @@
...
         />
 
+        <NodeListItem
+          v-for="(url, i) in customNodeURLs"
+          :key="i"
+          :url="url"
+          @refresh="loadURLs()"
+        />
+
         {{ '' && 'To Do: Render other saved networks from electron storage here' }}
```
Comment (alexandreaco) [inline src/views/Settings/SettingsSelectNode.vue:33]: We can remove this line now

**Review by alexandreaco — APPROVED** [review]
This works when I run it locally just like the original loom video. I left one comment about a line that can be removed. I'm going to base my work for [RDX-254](https://linear.app/township/issue/RDX-254/network-switching-remember-mainnet-and-stokenet) off of this work because they're tightly coupled

**Review by caleboller — CHANGES_REQUESTED** [review]
See the video below for more details, but I'd like to see the following minor changes:

- [x] clear the url string in the input when a custom node is added
- [ ] fix the delete node issue noted in the video
- [ ] optionally: require that custom node urls be unique (see video)

https://www.loom.com/share/d7ab399e0ef5488a9b14e1d4abd09867

**Comment by staylorwr** [comment]
https://www.loom.com/share/11d85e79b3d345e99478f3995efc1ce9

**Review by caleboller — APPROVED** [review]

**Pull request merged**